sed/loader/flash/loader.py

```diff
@@ -561,6 +730,76 @@ def fill_na(
         # Concatenate the filled dataframes
         return dd.concat(dataframes)
 
+    def parquet_handler(self, data_parquet_dir, **kwds):
```

Comment:
I would suggest to handle the parameters you list the same way we do in all other functions, i.e. list them as optional parameters with typing information = None, and handle them in the function accodingly, rather than using kwds as you do now. Or is there a strong reason doing this? Right now, there is no type checking done, and it is intransparent which parameters the function takes from the function prototype.